### DISCUSSION TITLE
Table sessions_v2_tmp_versioned purpose

### DISCUSSION THREAD

**Discussion by Tony-MF** [plausible/analytics#4728] [Self-Hosted Support] [answered]
Hi,

I'm trying to get some free space. I'm running v2.1.4.

<img width="479" alt="image" src="https://github.com/user-attachments/assets/283dfe1a-cc9c-49a2-978f-4ea8aabc1538">

What is the purpose of table sessions_v2_tmp_versioned, is it safe to delete it?

Thanks

**Comment by ruslandoga** [comment, marked as answer]
👋 @Tony-MF 

Yes, it's safe to be deleted. It's an artifact of https://github.com/plausible/analytics/blob/master/priv/ingest_repo/migrations/20240222082911_sessions_v2_versioned_collapsing_merge_tree.exs

**Reply by Tony-MF** [reply]
Thanks for the prompt reply!
